### DISCUSSION TITLE
Structuring elements in tab bar

### DISCUSSION THREAD

**Discussion by janschlatter** [mbnuqw/sidebery#338] [Ideas]
I have an idea to increase productivity even further, by implementing structuring elements to the tabs.

For example, I have a panel dedicated for equity trading. There are always tabs there which I need to access to research, or trade for example.

What would really help with keeping panels to specific tasks, would be:

- Make certain tabs uncloseable
- Colour certain tabs differently (propably easily done with a CSS class)
- Add seperators, for example like in the bookmark tab
- Add Titles for seperating. Would kinda look like a Group Tab but not act like it

What are your thoughts? I love love love this addon! Thanks so much for your work!

**Comment by noneuclideanmotion** [comment]
I love these ideas, they would be helpful for me also. :)
